### ISSUE TITLE
New update locked us out

### ISSUE THREAD

**Issue by cagrimertbakirci** [serghey-rodin/vesta#1738]
### Operating System (OS/VERSION):

CentOS 7

### VestaCP Version:

0.9.8-23 (security)

### Installed Software (what you got with the installer):

php, apache, nginx, mysql

### Steps to Reproduce:

The [most recent update](https://forum.vestacp.com/viewtopic.php?f=25&t=17785) caused us to get locked out of our root access.

In this file: /etc/ssh/sshd_config 

There is this line: PasswordAuthentication 

which we set to "No".

Also, we have: "#PermitRootLogin yes" in the same config file. 

It needs to be: "PermitRootLogin yes" 

Without these, it seems like we won't be able to access our root account, which prevents us from continue developing. We only have access to admin.

I know you guys cannot do much about this, but I was wondering if there is a way to roll the update back temporarily to fix these issues? Or what would you recommend?

**Comment by madeITBelgium** [comment]
You cannot login from admin account to root with this?
``` su - ```

**Comment by cagrimertbakirci** [comment]
> You cannot login from admin account to root with this?
> `su -`

Nope, that does not work, unfortunately. The reason is that the only user we had was given by Vesta Panel and it was only an admin. And since they removed the admin access to root with their most recent update, we were locked out.

But, we just found the solution to this problem: We use DigitalOcean as our server and luckily the Droplet's web console on DigitalOcean does not use SSH for it's connection; so any changes made to the SSH configuration will not apply to it's usage. 

Using this, we managed to regain access to root and from there, we could create a new user with root privileges. 

So if anyone had this problem, they can maybe use their server console to bypass the SSH problem.

Thanks for the help!
